### ISSUE TITLE
Ensure all audit warnings and failures are entered into GitHub

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#6071]
This issue will be considered done once all of the components with "warning" or "failure" have been entered into GitHub as issues.
These are the remaining components we have to add issues for:

- [x] Date picker 
- Mouse hover issue, [#5752](https://github.com/uswds/uswds-site/issues/2473)
- [x] Date range picker
- Mouse hover issue, [#5752](https://github.com/uswds/uswds-site/issues/2473)
- [x] Alert
- [ ] Site Alert
- [ ] Step indicator
- [ ] Card
- [ ] Modal
- [ ] Range slider
- [ ] Language selector
- Language selector could be more descriptive, [#6075](https://github.com/uswds/uswds/issues/6075) 
- [ ] Input mask
- [ ] Input prefix/suffix
